src/components/templates/Audience/Audience.scss

```diff
@@ -193,8 +205,9 @@ $seat-size: 4vh;
 
     .add-participant-button {
       color: black;
-      font-size: 3vh;
+      font-size: calc(#{$seat-size} * 0.75);
       align-self: center;
+      z-index: z(audience-participant-add-button);
```

Comment:
The buttons are overlayed by video-overlay and were not accessible

Reply:
Shouldn't the video overlay never sit over them though? When there are no `z-index`s set, it uses the 'element order' in the code, and the div for the overlay is before all of the seats. So the seats should be 'on top'.

Reply:
I don't think so. You can try removing this

Reply:
Yup, seems you are correct. I expect it's because it's `position: absolute`, so sits outside of the normal flow I was referring to.

A better solution than `z-index` for this case is to use `pointer-events: none` on `audience-overlay` though:

- https://css-tricks.com/almanac/properties/p/pointer-events/